test-d/foundry/foundry.js/pixi/shaders/abstractBaseShader.test-d.ts

```diff
@@ -1,5 +1,4 @@
 import { expectError, expectType } from 'tsd';
-import '../../../../index';
```

Comment:
isn't this needed anymore? We have this in a lot of test files if I'm not mistaken

Reply:
I removed it and the tests are still running. If you have a look at `test-d/foundry/common/documents.mjs/*` there are a lot of tests with `import '../../../../index'`, but ChatMessage, MeasuredTemplate and User don't have it. Why was it needed?

Reply:
No idea to be honest… maybe something was set up slightly different in the past. It’s fine then.